### PR TITLE
Travelogue

### PR DESCRIPTION
**Pull request by techlab-lainio**
- comparing this to `dev` branch even it wasn't there when started, is this OK?

### CODE REVIEW

**Comment by techlab-lainio** [comment]
we can change the base branch to `master` if the dev isn't OK. BTW, I really hope to get this out of my hand ASAP 🙈

File: content/en/blog/travelogue/index.md [hunk 1]
```diff
@@ -0,0 +1,421 @@
...
+- Headless WebAuthn authenticator 
+- React-based Web Wallet
+- Lots of documentation and examples
+- Agency's gRPC API v1
+- Polyglot implementations gRPC: TypeScript, Go, JavaScript
+- New toolbox both Web and Command-line 
+- The full OSS release 
+
+As said, all of the important elements are in place. However, our solution is
+based on `libindy`, which is interesting where the original contributor
```
Comment (lauravuo-techlab) [inline content/en/blog/travelogue/index.md:412]: I would reformat this somehow. _Our solution is currently based on libindy, which makes the future of the library interesting. Many from the Aries community are moving to shared libraries whereas the original contributor has chosen its own path._

**Review by lauravuo-techlab — APPROVED** [review]
Great overview of our project, especially the pics are superb!
I have added some comments and change suggestions for expressions or sections that I had trouble understanding when reading the first time around. You can use your own judgement if those are improvements or not 😄

File: content/en/blog/travelogue/index.md [hunk 2]
```diff
@@ -0,0 +1,421 @@
...
+
+### Technology Tree
+
+{{< imgproc tree.png Resize "1991x" >}}
+<em>Findy Agency Tech Tree</em>
+{{< /imgproc >}}
+
+Technology roots present the most important *background knowledge* of our study.
+The most fundamental technologies and study subjects are in the trunk.  The
+trunk is the backbone of our work. It tights altogether. Branches and leaves
```
[the remark this answers is not in the document]
Comment (techlab-lainio) [reply content/en/blog/travelogue/index.md:38]: This was so funny :-D, but it seems that this is finally the version that is most formally correct: it ties it all together.

File: content/en/blog/travelogue/index.md (hunk 1 above)
Comment (techlab-lainio) [reply content/en/blog/travelogue/index.md:412]: nice, and totally agree. actually moving a little bit back to the original, maybe I trusted too much Grammarly on this :-)

**Pull request merged**